oslo/data/serialize_corpora.py

```diff
@@ -0,0 +1,87 @@
+from typing import Optional
+from dataclasses import dataclass, field
+
+from transformers import HfArgumentParser
+
+from src.processors import GPT2Processer
+from src.utils import load_corpora
+
+model_type_to_processor = {
+    "gpt2": GPT2Processer,
+}
+
+
+@dataclass
+class Arguments:
```

Comment:
same. I think this class should not be included under `tests` directory of root.